src/vmspawn/vmspawn-util.c

```diff
@@ -236,3 +253,35 @@ int find_qemu_binary(char **ret_qemu_binary) {
 
         return find_executable(qemu_arch_specific, ret_qemu_binary);
 }
+
+int machine_cid(unsigned int *ret_machine_cid) {
```

Comment:
Coding style: we write `unsigned` instead of `unsigned int`

Reply:
Fixed in 084ed8ddc7.